week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -68,7 +67,18 @@ function checkCodeIsThere(stringText) {
   
   Hint: Use the corresponding array method to split the array.
 */
-function getTransportModes() { }
+
+function getTransportModes(arraysOfLocAndTransport) {
+  let transportMode = arraysOfLocAndTransport.slice(1);
+  return transportMode;
+}
+
+//AND
+
+function getTransportModes(transportMode) {
+  let transport = transportMode.splice(1, 2);
+  return transport;
+}
 
```

Comment:
We tend not to use `splice` there are reason behind this.

https://www.freecodecamp.org/news/lets-clear-up-the-confusion-around-the-slice-splice-split-methods-in-javascript-8ba3266c29ae/

Thanks for showing you can use both. EXCELLLLENT WORK!